solr/core/src/test/org/apache/solr/search/TestExtendedDismaxParser.java

```diff
@@ -415,19 +415,7 @@ public void testFocusQueryParser() {
     assertQ(req("defType","edismax", "mm","0", "q","Terminator: 100", "qf","movies_t foo_i"),
             twor);
 
-    assertQ(req("defType","edismax", "mm","100%", "q","Terminator: 100", "qf","movies_t foo_i", "sow","true"),
-            nor);
-    // When sow=false, the per-field query structures differ (no "Terminator" query on integer field foo_i),
-    // so a dismax-per-field is constructed.  As a result, mm=100% is applied per-field instead of per-term;
-    // since there is only one term (100) required in the foo_i field's dismax, the query can match docs that
-    // only have the 100 term in the foo_i field, and don't necessarily have "Terminator" in any field.
```

Comment:
@sarowe, the above comment (from dd171ff8fe31df578b7e6fab1eb5bfc1ade3f5fc) describes behavior that seems counterintuitive, but it doesn't give any indication of whether the behavior might be considered desirable for some reason? The behavior described would be changed by this PR; this seems like it would be an improvement, but I wanted to call your attention to this comment specifically, just to make sure you're aware/supportive of the change.